.github/workflows/tests.yml

```diff
@@ -0,0 +1,24 @@
+name: tests
+
+on:
+  pull_request:
+      types: [ opened, synchronize, reopened, ready_for_review, labeled, unlabeled ]
+  push:
+    branches: [ master, 276-tests-with-github-actions]
+    
+jobs:
+
+  tests:
+    runs-on: ubuntu-latest
+    container: rust:1.53.0
+    env:
+      RUST_TC_VER: nightly-2021-06-23
+    steps:
+      - name: Checks-out repository 
+        uses: actions/checkout@v2
+      - name: Tests
+        run: |
+          apt-get update && apt-get install -y --no-install-recommends musl-tools
+          rustup install ${{ env.RUST_TC_VER }}
+          rustup default ${{ env.RUST_TC_VER }}
+          cargo test
```

Comment:
I found an action for `cargo`: https://github.com/marketplace/actions/rust-cargo
And also an action to setup rust: https://github.com/marketplace/actions/rust-toolchain

Can we try to use that actions?

Reply:
It looks like `actions-rs/cargo@v1` can also be used here